src/data/faq.json

```diff
@@ -493,10 +493,11 @@
                         "active": true,
                         "textblock": [
                             "If you see such a notice, your device is not able to send any random IDs via Bluetooth. This can be caused by a restriction due to your combination of hardware and software on your device. Because of it, the Corona-Warn-App cannot use Bluetooth Low Energy in its so-called “peripheral mode”.",
-                            "In case you are infected and you decide to warn others, only users who were checked in to the same event or the same location, at the same time as you were, can be warned. Your own risk assesment is <b>not affected</b>. All other features of the app are also not affected.",
+                            "In case you are infected and you decide to warn others, only users who were checked in to the same event or the same location, at the same time as you were, can be warned. Your own risk assessment is <b>not affected</b>. All other features of the app are also not affected.",
                             "Please ensure that you are using the latest version of the operating system available for your smartphone. If the problem persists, please contact your device manufacturer.",
                             "If the card shows that your device is completely incompatible, the cause could be that your device does not contain a Bluetooth chip. Only in this case, you cannot exchange random IDs with others and risk assessment can only base its calculation on locations and events you checked in to.",
-                            "<b>Both Check-in and Contact Journal functionality, as well as retrieving PCR and rapid test results, is fully functional without restriction in any case</b> and only proximity-based ID recording is affected."
+                            "<b>Both Check-in and Contact Journal functionality, as well as retrieving PCR and rapid test results, is fully functional without restriction in any case</b> and only proximity-based ID recording is affected.",
+                            "Bluetooth compatibility checking and warnings were introduced with Android Corona-Warn-App version 2.2."

```

Comment:
This sounds a bit like Bluetooth warnings were added with this version (if you read "compatibility checking" as one unit)

Reply:
I started out with a more detailed description and then I simplified it, so I hope that the simplification doesn't cause any confusion! I was a bit surprised that the PR got merged so quickly, but now that it is merged, I would leave it like it is, unless it is going to cause problems.